packages/stark-build/config/webpack.prod.js

```diff
@@ -83,7 +106,7 @@ module.exports = function() {
 			 *
 			 * See: http://webpack.github.io/docs/configuration.html#output-path
 			 */
-			path: helpers.root(buildUtils.getAngularCliAppConfig().outDir),
+			path: helpers.root(buildUtils.getAngularCliAppConfig()["architect"]["build"]["options"]["outputPath"]),
```

Comment:
Again, I think we should just use the dot syntax whenever possible...

Reply:
done